Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -50,6 +48,7 @@ public func verifySnapshot<A, B>(
       let snapshotDirectoryUrl: URL = directoryUrl
         .appendingPathComponent("__Snapshots__")
         .appendingPathComponent(fileName)
+      let testName = testName.hasSuffix("()") ? String(testName.dropLast(2)) : testName
```

Comment:
The ideal solution here might be to do some sort of regex and rip out the entire thing, but given that test functions normally take no parameters I figured this would be a rare case.